### PR TITLE
generate controllers and views

### PR DESCRIPTION
**Pull request by Gardimy**
For this milestone, I just created a basic controllers and views.

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -21,7 +21,11 @@ default: &default
 
 development:
   <<: *default
-  database: esport_gaming_center_backend_development
+  database: gaming
+  username: postgres
+  password: gardy
```
Comment (alvp01) [inline config/database.yml:26]: Changes a required to use a general DB password (environment variables)

File: app/views/layouts/games/inde.html.erb
Comment (alvp01) [inline app/views/layouts/games/inde.html.erb:1]: View files not required, please delete them

**Review by alvp01 — CHANGES_REQUESTED** [review]
Some changes are required

**Comment by lerfast** [comment]
### Hi  @Gardimy  👋 👋,

This PR is completed! There is nothing else to say other than... it's time to merge it 🚀👨🏼‍🚀
Congratulations! 🎉

### STATUS: APPROVED ✔✔

nicejob

🎯 TOP HIGHLIGHTS 🎯
- Cleanly written code ✅
- Working as expected ✅

**Pull request merged**